charts/openmetadata/values.yaml

```diff
@@ -149,6 +149,23 @@ global:
   #   clientId: "<client_id>"
   #   callbackUrl: "<callback_url>"
 
+  # Example LDAP Auth Config
+  # authentication:
+  #   provider: "ldap"
+  #   publicKeys:
+  #   - "http://openmetadata:8585/api/v1/config/jwks"
+  #   authority: "https://accounts.google.com"
+  #   clientId: "<client_id>"
+  #   callbackUrl: "<callback_url>"
+  #   enableSelfSignup: false
+  #   ldapConfiguration:
+  #     host: "example.ldap.server.com"
+  #     port: 10636
+  #     dnAdminPrincipal: "cn=admin,dc=example,dc=com"
+  #     dnAdminPassword: "secret"
+  #     userBaseDN: "ou=people,dc=example,dc=com"
+  #     mailAttributeName: email
```

Comment:
I thought it will be confusing if default values for ldapConfiguration are provided even if the provider is not equal to `"ldap"`. 
But as you pointed out, since those values are rendered into `env` only when provider is set to "ldap", I think it will be okay to have them as default value, to enable users to use default values :)